jquery.canvaswrapper.js

```diff
@@ -528,6 +530,211 @@
+     * 
+     * @constructor
+     * @param {string} cls The class name of the element
+     * @param {element} container Element onto which to append the canvas.
+     */
+    var WebGlCanvas = function(cls, container) {
+        var element = container.getElementsByClassName(cls)[0];
+        var renderer, camera,
+            scenes = [new THREE.Scene()],
+            mainscene = new THREE.Scene();
+
+        if (!element) {
+            element = document.createElement('canvas');
+            element.className = cls;
+            element.style.direction = 'ltr';
+            element.style.position = 'absolute';
+            element.style.left = '0px';
+            element.style.top = '0px';
+
+            container.appendChild(element);
+
+            // If HTML5 Canvas isn't available, throw
+            if (!element.getContext) {
+                throw new Error('Canvas is not available.');
+            }
+        }
+        renderer = new THREE.WebGLRenderer({canvas: element, antialias: false, alpha: true });
+
+        // Determine the screen's ratio of physical to device-independent
+        // pixels.  This is the ratio between the canvas width that the browser
+        // advertises and the number of pixels actually present in that space.
+
+        // The iPhone 4, for example, has a device-independent width of 320px,
+        // but its screen is actually 640px wide.  It therefore has a pixel
+        // ratio of 2, while most normal devices have a ratio of 1.
+
+        var devicePixelRatio = window.devicePixelRatio || 1,
+            backingStoreRatio = 1,
+            box = container.getBoundingClientRect(),
+            rendererSize = { width: box.width, height: box.height };
+
+        // Size the canvas to match the internal dimensions of its container
+        this.width = box.width;
+        this.height = box.height;
+
+        this.element = element;
+        this.scenes = scenes;
+        this.mainscene = mainscene;
+        this.camera = camera = new THREE.OrthographicCamera(this.width / 2, -this.width / 2, -this.height / 2, this.height / 2, 0.1, 1000);
+        this.renderer = renderer;
+        if (renderer) {
+            backingStoreRatio =
+            renderer.webkitBackingStorePixelRatio ||
+            renderer.mozBackingStorePixelRatio ||
+            renderer.msBackingStorePixelRatio ||
+            renderer.oBackingStorePixelRatio ||
+            renderer.backingStorePixelRatio || 1;
+
+            this.pixelRatio = devicePixelRatio / backingStoreRatio
+            mainscene.userData.pixelRatio = this.pixelRatio;
+            mainscene.userData.rendererSize = rendererSize;
+            renderer.setPixelRatio(this.pixelRatio);
+            renderer.setSize(this.width, this.height, true);
+            renderer.autoClear = true;
+            camera.aspect = this.width / this.height;
+            this.cameraSight = new THREE.Vector3(this.width / 2, this.height / 2, 1000);
```

Comment:
Set camera position to the middle of the scene at z = 0.
Set camera view point to the middle of the scene at z = 1000.